### PR TITLE
Parameterize ES index name by stage

### PR DESCRIPTION
**Pull request by kislyuk**
This is needed to be able to run staging and dev using the same ES cluster, which I'd like to keep doing for now. (Prod will get its own cluster)

Connected to #232

### CODE REVIEW

File: dss/__init__.py [hunk 1]
```diff
@@ -26,14 +26,14 @@
 # CONSTANTS COMMON TO THE INDEXER AND QUERY ROUTE.
 
 # ES index containing all docs
-DSS_ELASTICSEARCH_INDEX_NAME = "hca"
+DSS_ELASTICSEARCH_INDEX_NAME = "hca-" + os.environ["DSS_DEPLOYMENT_STAGE"]
```
Comment (mikebaumann) [inline dss/__init__.py]: Because the DSS is intended to be project independent, we should use a more general index base name than "hca". Suggestions are welcome. "index"?,  "primary"? ...
Yet, that may be best done as a different PR.

Comment (kislyuk) [reply dss/__init__.py]: yeah I thought that too. "dss-docs" might do.
